config/open-feature/sdk-dart/workgroup.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+  - dart-sdk-server
```

Comment:
This repo name is a proposal.

Reply:
How about `dart-server-sdk`?

Reply:
Yes sounds even better, I will rename it 👌